### PR TITLE
Add schedule

### PR DESCRIPTION
**Pull request by dns2utf8**
* [x] Fix the blog loading on too many pages
* [x] Add schedule
    * [ ] add @skade 
    * [ ] add @zeenix

### CODE REVIEW

**Comment by dns2utf8** [comment]
Please check your links and descriptions

Bildschirmfoto am 2019-10-23 um 22 45 45

File: _layouts/default.html [hunk 1]
```diff
@@ -11,40 +11,6 @@
...
-                <article>${description.substr(0, 350)} ...
-                <a href="${url}">continue reading</a></p>
-                </article>
-              </p>
-            `;
-            document.querySelector('#news .post').innerHTML = postHtml;
-          });
-        })
-      });
-    </script>
```
Comment (dns2utf8) [inline _layouts/default.html:47]: Don't load the blog posts on all pages except the `index.html`

**Comment by RafaelaWetzel** [comment]
Florian's last name is written wrong, it should be "Gilcher"

**Comment by dns2utf8** [comment]
> Florian's last name is written wrong, it should be "Gilcher"

Thank you, it is fixed

**Comment by badboy** [comment]
We got a new picture from Zeeshan, we should swap it out

File: schedule.md [hunk 2]
```diff
@@ -26,7 +26,10 @@ This will allow us to have a less packed schedule and more time for you to enjoy
 
 ### Exact schedule
```
Comment (badboy) [inline schedule.md]: Let's remove the whole block of text above this (and this heading).
Let's also add the links to the talks and workshop overview as we have that on the talks and workshop sites itself.

**Review by badboy — CHANGES_REQUESTED** [review]

**Comment by badboy** [comment]
Oh. and I swapped out the picture

**Review by badboy — APPROVED** [review]

**Pull request merged**